<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KvizLogin</class>
 <widget class="QDialog" name="KvizLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../ipi.jpg</normaloff>../ipi.jpg</iconset>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>13</y>
     <width>225</width>
     <height>117</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lblUser">
        <property name="text">
         <string>Korisniƒçko ime:</string>
        </property>
        <property name="buddy">
         <cstring>txtUser</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtUser">
        <property name="minimumSize">
         <size>
          <width>141</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lblPass">
        <property name="text">
         <string>Lozinka:</string>
        </property>
        <property name="buddy">
         <cstring>txtPass</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtPass">
        <property name="minimumSize">
         <size>
          <width>171</width>
          <height>22</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btnLogin">
        <property name="minimumSize">
         <size>
          <width>221</width>
          <height>31</height>
         </size>
        </property>
        <property name="text">
         <string>Prijava</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblResult">
        <property name="minimumSize">
         <size>
          <width>221</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string>Status...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
